pkg/utils/utils.go

```diff
@@ -85,15 +85,7 @@ func GetPfName(pciAddr string) (string, error) {
 	files, err := ioutil.ReadDir(path)
 	if err != nil {
 		if os.IsNotExist(err) {
-			path := filepath.Join(sysBusPci, pciAddr, "net")
-			files, err = ioutil.ReadDir(path)
-			if err != nil {
-				return "", err
-			}
-			if len(files) < 1 {
-				return "", fmt.Errorf("no interface name found for device %s", pciAddr)
-			}
-			return files[0].Name(), nil
+			return "", nil
```

Comment:
The comment for this function need to change accordingly.

Reply:
done thanks!